scipy/stats/stats.py

```diff
@@ -3999,6 +3999,52 @@ def pearsonr(x, y):
     >>> stats.pearsonr([1, 2, 3, 4, 5], [10, 9, 2.5, 6, 4])
     (-0.7426106572325057, 0.1505558088534455)
 
+    It is important to keep in mind that no correlation does not imply
+    independence unless (X, Y) is Gaussian. Correlation can even be zero
+    when there is a very simple dependence structure: if X follows a
+    standard normal distribution, let Y = abs(X). A simple check
+    confirms that the correlation is close to zero:
+
+    >>> x = stats.norm.rvs(size=500)
+    >>> y = np.abs(x)
+    >>> stats.pearsonr(x, y)
+    (-0.016172891856853524, 0.7182823678751942) # may vary
+
+    Indeed, since the expectation of x is zero, cov(X, Y) = E(X*Y). This equals
+    EX*abs(X) which is zero by symmetry.
```

Comment:
```suggestion
    E[X]*abs(X) which is zero by symmetry.
```
I think?

Reply:
it shoud be E[X*abs(X)], I will change it